relay-event-schema/src/protocol/security_report.rs

```diff
@@ -442,6 +481,26 @@ struct CspReportRaw {
     csp_report: CspRaw,
 }
 
+/// Defines CSP report sent through the [Reporting API](https://developer.mozilla.org/en-US/docs/Web/API/Reporting_API).
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct CspViolationRaw {
+    #[serde(rename = "type")]
+    ty: String,
+    body: CspRaw,
+
+    /// For forward compatibility.
```

Comment:
The field seems unused and the struct is also never serialized, seems like that does nothing and can be removed.